src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_SIZE = 16;
+    private static final int GROW_FACTOR = 2;
+    private static int size;
+    private int capacity;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_SIZE];
+        size = DEFAULT_SIZE;
+    }
 
     @Override
     public void put(K key, V value) {
+        resize();
+        int index = hash(key);
+        Node<K,V> newNode = new Node<>(key, value);
+        Node<K,V> current = table[index];
+        if (current == null) {
+            table[index] = newNode;
+        } else {
+            current = findNodeInCollisium(key);
+            if (Objects.equals(current.key, key)) {
+                current.value = value;
+                return;
+            } else {
+                current.next = newNode;
+            }
+        }
+        this.capacity++;
```

Comment:
let's only use this where it's necessary
https://stackoverflow.com/a/2411283
```suggestion
        size++;
```